enhancements/service-catalog/service-catalog-and-broker-removal.md

```diff
@@ -0,0 +1,187 @@
+---
+title: service-catalog-and-broker-removal
+authors:
+  - "@jmrodri"
+reviewers:
+  - "@dmesser"
+  - "@joelanford"
+  - "@robszumski"
+approvers:
+  - TBD
+creation-date: 2019-12-17
+last-updated: 2019-12-17
+status: implementable
+see-also:
+  - "/enhancements/service-catalog/prepare-service-catalog-and-brokers-for-removal.md"
+replaces:
+superseded-by:
+---
+
+# Service Catalog and Broker Removal
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Open Questions [optional]
+
+This is where to call out areas of the design that require closure before deciding
+to implement the design.  For instance, 
+ > 1. This requires exposing previously private resources which contain sensitive
+  information.  Can we do this? 
+
+## Summary
+
+In OpenShift 4.1, we announced the deprecation of the Service Catalog, the
+Ansible Service Broker, and the Template Service Broker. In 4.4, we plan to
+remove them from productization. They will no longer be installed in a new
+cluster. They will also block cluster upgrades until they are disabled in older
+clusters.
+
+## Motivation
+
+Service Catalog and Service Brokers need to be removed in OpenShift 4.4 because
```

Comment:
4.4 is not necessarily the LTS, the motivation is to just finish the previously announced deprecation.

Reply:
Removed reference to LTS.